CHANGELOG.txt

```diff
@@ -9,6 +9,7 @@
   Users side :
   ------------
     # The version of Django has been upgraded to "3.1".
+    # The version of jQuery has been upgraded to "1.12.4"
```

Comment:
You can remove this one (django is in users' section because of installation/virtual env)